src/SDKs/Search/DataPlane/Search.Tests/Tests/AutocompleteTests.cs

```diff
@@ -172,7 +172,7 @@ protected void TestAutocompleteWithSelectedFields()
             SearchIndexClient client = GetClientForQuery();
             var autocompleteParameters = new AutocompleteParameters()
             {
-                SearchFields = "hotelName"
+                SearchFields = new[] { "hotelName" }
```

Comment:
We need tests with more than one search field.

Reply:
Added one more